.gitignore

```diff
@@ -0,0 +1,17 @@
+# Binaries for programs and plugins
+*.exe
+*.dll
+*.so
+*.dylib
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+.glide/
+.vscode/
+bin/
+caimake/caimake.sh
```

Comment:
why do you ignore this?  IIUC, it's put under `/hack/caimake/caimake.sh`.

Reply:
`/hack/caimake/caimake.sh` MUST be generated every time when building.
see more imformation in `Makefile.expansion`